### PR TITLE
fix(dashboard): remove tailing digits on widget dragging

### PR DESCRIPTION
**Pull request by square-li**
## Overview
uses the same `trimWidgetPosition` function from `moveWidgetAction` to trim position & size numbers.

also add tests for this behaviour.
## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: packages/dashboard/src/components/sidePanel/sections/baseSettingSection/index.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import React, { FC } from 'react';
 import { DashboardMessages } from '~/messages';
 import { useInput } from '../../utils';
-import { NonCancelableEventHandler } from '@cloudscape-design/components/internal/events';
```
Comment (square-li) [inline packages/dashboard/src/components/sidePanel/sections/baseSettingSection/index.tsx:4]: remove using internal types

**Review by jmbuss — COMMENTED** [review]
LGTM, what happens if someone enters a decimal into the input widget? do we allow that and then trim after they're done typing?

**Comment by square-li** [comment]
> LGTM, what happens if someone enters a decimal into the input widget? do we allow that and then trim after they're done typing?
 
No we don't allow that. All decimal will be trimmed after typing. 
chrome-capture-2023-1-22 (1)

File: packages/dashboard/src/components/sidePanel/sections/baseSettingSection/index.spec.tsx [hunk 2]
```diff
@@ -32,9 +39,28 @@ it('renders', () => {
...
+  expect(wrapper.findInput('[data-test-id="base-setting-height-input"]')?.findNativeInput().getElement().value).toBe(
+    '11'
+  );
+});
+
+it('rounds input on user input', async () => {
+  const wrapper = createWrapper(render(<TestBaseSettingSection />).baseElement);
+  const xInput = wrapper.findInput('[data-test-id="base-setting-x-input"]')!.findNativeInput().getElement();
+  await userEvent.type(xInput!, '2.4'); // current x is 2, this type appends 2 after it.
+  expect(xInput?.value).toBe('22'); // decimal digits on typing will be ignored not rounded.
```
Comment (jmbuss) [inline packages/dashboard/src/components/sidePanel/sections/baseSettingSection/index.spec.tsx:65]: shouldn't this be just 2? or would it end up being 24?

Comment (square-li) [reply packages/dashboard/src/components/sidePanel/sections/baseSettingSection/index.spec.tsx:65]: Current X for the widget is `2` and `type` event just append new chars after it.  So it is `2` -> `22.4` -> `22`

Comment (jmbuss) [reply packages/dashboard/src/components/sidePanel/sections/baseSettingSection/index.spec.tsx:65]: Oh I see, so the idea is you're attempting to change the x from 2 -> 22.4

I thought it was a scenario trying to change 2 -> 2.4

perfect lgtm

**Review by jmbuss — APPROVED** [review]

**Pull request merged**